RCAIDE/Framework/Analyses/Analysis.py

```diff
@@ -0,0 +1,226 @@
+# RCAIDE/Framework/Analyses/Analysis.py
+# (c) Copyright 2023 Aerospace Research Community LLC
+# 
+# Created:  Jun 2024, M. Clarke 
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------  
+from RCAIDE.Framework.Core import Container as ContainerBase
+from RCAIDE.Framework.Core import Data
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Analysis
+# ----------------------------------------------------------------------------------------------------------------------   
+class Analysis(Data):
+    """ The top level Analysis class 
+    """
+    def __defaults__(self):
+        """This sets the default values and methods for the analysis.
+        
+            Assumptions:
+                None
+    
+            Source:
+                None
+    
+            Args:
+                None
+    
+            Returns:
+                None 
+            """           
+        self.tag    = 'analysis'
+        self.features = Data()
+        self.settings = Data() 
+        
+    def initialize(self,*args,**kwarg):

```

Comment:
This needs to disappear, if we do mission pre_process

Reply:
Done

Reply:
To clarify, this is remaining so is mission pre_process no longer happening?

Reply:
Resolved by elimination of pre_process step